### PR TITLE
Add unit and sanity GitHub workflows

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

This PR adds GitHub Workflows to run linters, changelog, sanity and unit test jobs.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by gravesm — APPROVED** [review]

File: galaxy.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 namespace: vmware
 name: vmware_rest
 readme: README.md
-version: null
+version: 2.3.2-dev0
```
Comment (alinabuzachis) [inline galaxy.yml:4]: Shouldn't it be 2.3.1?

Comment (GomathiselviS) [reply galaxy.yml:4]: We have to decide what version to add to the collections. I see *-dev0 version added to amazon.aws. @jillr @gravesm your thoughts, please?

Comment (mariolenz) [reply galaxy.yml:4]: I set it to the correct version in [community.vmware](https://github.com/ansible-collections/community.vmware) when I release and set it back to `null` afterwards: [History for community.vmware/galaxy.yml](https://github.com/ansible-collections/community.vmware/commits/main/galaxy.yml)

Don't know if this helps.

ansible-collections/community.vmware#1320

**Comment by mariolenz** [comment]
I've added at least sanity test for 2.13, 2.14 and 2.15 (from the [collection_template](https://github.com/ansible-collections/collection_template)) in #431.

Since it looks like there are no unit tests in this collection, I wonder if you might consider closing this PR as resolved by my PR.

**Comment by mariolenz** [comment]
@GomathiselviS FYI we now also have a GHA linting workflow: #410

BTW do _you_ have an idea why ansible/ansible-zuul-jobs#1826 started to fail the CI a couple of days ago? I don't understand it, the first ansible/check succeeded just fine :-/

**Comment by GomathiselviS** [comment]
Workflows added

**Pull request closed without merging**